src/main/groovy/au/org/ala/spatial/process/SlaveProcess.groovy

```diff
@@ -677,12 +678,13 @@ class SlaveProcess {
         }
 
         String layerPath = standardLayersDir + File.separator + resolution + File.separator + layer
+        taskLog("Loading " + layerPath)
```

Comment:
Do not log the full local path

Reply:
Aha, I am meant to change, but I forgot. Will do again